packages/terra-site/src/examples/embedded-component/Index.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Didn't realize that terra-framework's existing site package is `terra-framework-site`. We should move this example to the `terra-framework-site` package. In the mean time, just created this [issue](https://github.com/cerner/generator-terra-module/issues/78).

Reply:
Yeah, we'll want to move this to `terra-framework-site`. Copy / paste `embedded-component/Index.jsx` into `packages/terra-framework-site/src/examples/` and then this is good to go.